PrimerSDK/Classes/User Interface/Primer/PaymentMethodComponent.swift

```diff
@@ -7,6 +7,7 @@
 
 import UIKit
 
+// FIXME: Unused?
 class PaymentMethodComponent: UIView {
```

Comment:
if not used anywhere at the moment, let's remove

Reply:
I was mistaken about this one